src/test/java/stepDefinition/ContactUsSteps.java

```diff
@@ -23,4 +24,23 @@ public void userValidatesThatContactUsPageIsOpen() {
         WebElement contactUsPage = driver.findElement(ContactUsPage.ContactUsHeading);
         Assert.assertTrue(contactUsPage.isDisplayed());
     }
+
+    @When("User clicks on input field {string} and enters {string}")
```

Comment:
user enters..... no need to mention that there is a click first

Reply:
Done